package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uswds-docs",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
actually yeah that's a good question. Up until now I think the site has just been matching versions with the standards since we'd tend to push updates to both at the same time but I'm thinking we should probably figure out a better pattern for versioning the site going forward (like does a content update constitute a version bump?)